src/middlewared/middlewared/plugins/update_/trains.py

```diff
@@ -1,54 +1,62 @@
-# -*- coding=utf-8 -*-
-import aiohttp
-import async_timeout
+import json
+
+from aiohttp import ClientSession, ClientTimeout
 
 from middlewared.service import private, Service
 from middlewared.utils.network import INTERNET_TIMEOUT
-
-from .utils import can_update, scale_update_server
+from .utils import can_update, scale_update_server, SCALE_MANIFEST_FILE
 
 
 class UpdateService(Service):
+    opts = {'raise_for_status': True, 'trust_env': True, 'timeout': ClientTimeout(INTERNET_TIMEOUT)}
+    update_srv = scale_update_server()
+
     @private
-    async def get_scale_trains_data(self):
-        async with async_timeout.timeout(INTERNET_TIMEOUT):
-            async with aiohttp.ClientSession(
-                raise_for_status=True, trust_env=True,
-            ) as session:
-                trains = await (await session.get(f"{scale_update_server()}/trains.json")).json()
+    def get_manifest_file(self):
+        with open(SCALE_MANIFEST_FILE) as f:
```

Comment:
I think you can read it on module load too (using `pathlib`), it does not change during system runtime